flang/test/Lower/OpenMP/teams.f90

```diff
@@ -1,6 +1,6 @@
 ! REQUIRES: openmp_runtime
```

Comment:
nit: I don't think this should require openmp_runtime. We are only emitting hlfir.

Reply:
It's because of the `omp_high_bw_mem_alloc` in the allocate clause (in `teams_allocate`).